examples/regular_polygons.py

```diff
@@ -0,0 +1,89 @@
+import random
+import sys
+
+import geometry
+import pygame as pg
+
+FPS = 60
+RES = WIDTH, HEIGHT = 640, 480
+DARK_GREY = (10, 10, 10)
+SIDES = [3, 4, 5, 6, 7, 8, 9, 10, 11, 12]
+COLORS = {
+    3: (255, 0, 0),
+    4: (0, 255, 0),
+    5: (0, 0, 255),
+    6: (255, 255, 0),
+    7: (0, 255, 255),
+    8: (255, 0, 255),
+    9: (255, 255, 255),
+    10: (255, 127, 0),
+    11: (127, 0, 255),
+    12: (0, 255, 127),
+}
+
+
+class Game:
+    def __init__(self):
+        pg.init()
+        # pg.mouse.set_visible(False)
+        self.display = pg.display
+        self.screen = self.display.set_mode(RES)
+        self.clock = pg.time.Clock()
+        self.colors = COLORS
+        self.radius = 64
+        self.toggle_mouse_button = False
+        self.polygons = []
+
+    def events(self):
+        for event in pg.event.get():
+            if event.type == pg.QUIT or (event.type == pg.KEYDOWN and event.key == pg.K_ESCAPE):
+                print("\nQuitting program...")
+                pg.quit()
+                sys.exit()
+            if pg.mouse.get_pressed()[0]:
```

Comment:
There's another way to do this, try using `pygame.MOUSEBUTTONDOWN` instead.

Reply:
I didn't know that. I'll use that variable

Reply:
Using `pygame.MOUSEBUTTONDOWN` sometimes has glitches when you click the screen. I added a second condition ` if pg.MOUSEBUTTONDOWN and any(pg.mouse.get_pressed())`

Reply:
I'll take a further look at this in the near future, for now I'll keep this as unresolved.